File: day18.java


public class Solution {
    // Write your code here.
    Stack<Character> stacky;
    Queue<Character> queuey;

    public Solution(){
        stacky = new Stack<Character>();
        queuey = new LinkedList<> ();
    }

    public void pushCharacter(Character ch){
        stacky.push(ch);
    }
    public void enqueueCharacter(Character ch){
        queuey.add(ch);
    }
     public Character popCharacter(){
        return stacky.pop();
    }
    public Character dequeueCharacter(){
        return queuey.remove();
    }

File: day18.py


class Solution:
    # Write your code here
    def __init__(self):
        super().__init__()
        self.queuey=[]
        self.stacky=[]

    def pushCharacter(self, ch):
        self.stacky.append(ch)
    def enqueueCharacter(self, ch):
        self.queuey.append(ch)

    def popCharacter(self):
        return self.stacky.pop()


    def dequeueCharacter(self):
        return self.queuey.pop(0)
